reference/specs/bindings/twitter.md

```diff
@@ -1,23 +1,24 @@
 # Kubernetes Events Binding Spec
 
-```yml
+```yaml
 apiVersion: dapr.io/v1alpha1
 kind: Component
 metadata:
-  name: <name>
+  name: <NAME>
+  namespace: <NAMESPACE>
 spec:
   type: bindings.twitter
   metadata:
   - name: consumerKey
-    value: ''
+    value: ""
```

Comment:
Other bindings have examples inline. Are there examples to help people understand how to use this binding?

Reply:
done